tests/test_validator.py

```diff
@@ -77,6 +77,8 @@ def test_validation_fails_when_model_has_unknown_root(self):
         assert_errors_exist(errors)
         assert_errors_contain(errors, "unrecognized.*root.*[a]")
 
+    # Disabling this test for now -- it's failing due to pollution from other tests and the aac_spec plugin.
+    '''
```

Comment:
If we're ignoring this test, it would be better to skip it using:
```python
from unittest import skip
# ...
@skip("This is failing due to pollution from other tests and the aac_spec plugin")
def test_data_validation(self):
    # ...
```

Reply:
I learned something new about python unittest today!  Thanks!